.github/stale.yml

```diff
@@ -2,13 +2,20 @@
 
 # Issues or Pull Requests with these labels will never be
 # considered stale. Set to `[]` to disable
----
+
 exemptLabels:
   - keepalive
   - security
   - reliability
   - release requirement
 
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
This value is configured on both issue https://github.com/ceph/ceph-csi/blob/a2c108159172c68e9b66c9844b5c526451db9439/.github/stale.yml#L33-L34 and PR https://github.com/ceph/ceph-csi/blob/a2c108159172c68e9b66c9844b5c526451db9439/.github/stale.yml#L44-L45  and this configuration is working fine in rook https://github.com/rook/rook/blob/master/.github/stale.yml @travisn or @leseb can you please confirm